### PR TITLE
Support obfuscating cstr literals

### PR DESCRIPTION
**Pull request by CasualX**

### CODE REVIEW

File: src/lib.rs [hunk 1]
```diff
@@ -199,10 +200,20 @@ pub mod words;
 
 #[doc(hidden)]
 #[inline(always)]
-pub fn unsafe_as_str(bytes: &[u8]) -> &str {
+pub const fn unsafe_as_str(bytes: &[u8]) -> &str {
 	// When used correctly by this crate's macros this should be safe
 	#[cfg(debug_assertions)]
-	return str::from_utf8(bytes).unwrap();
+	return match str::from_utf8(bytes) { Ok(s) => s, Err(_) => panic!("invalid str") };
```
Comment (PaulDance) [inline src/lib.rs:206]: Just passing by, even if this is only internal: the `const` part should not be required in both cases anymore, right?

**Review by PaulDance — COMMENTED** [review]
The rest looks good, though! Maybe a test somewhere? Also, adding to the existing `examples`?

**Comment by PaulDance** [comment]
Any chance to merge and release this some time soon?

**Comment by CasualX** [comment]
I haven't been very active sorry, I'll finish it this weekend.

**Pull request merged**